zutil.c

```diff
@@ -20,18 +20,18 @@ z_const char * const PREFIX(z_errmsg)[10] = {
     (z_const char *)""
 };
 
-const char zlibng_string[] =
-    " zlib-ng 2.1.0.devel forked from zlib";
-
 #ifdef ZLIB_COMPAT
 const char * Z_EXPORT zlibVersion(void) {
     return ZLIB_VERSION;
 }
-#endif
+#else
+const char zlibng_string[] =
```

Comment:
This string is not used anywhere - only exported. Not sure if we really need it imo.

Reply:
It makes sure we have that plaintext string in the resulting binary, thus on linux we can run `strings libz-ng.so` and it will show that string. It can therefore be used for simple identification of version and potentially detection of zlib-ng being included in third-party applications along with what version they compiled in.

Reply:
In compatibility mode, we could use something like `zlibng_compat_string`, this would avoid conflict when mixing native zlib-ng and compatibility library, and also mixing zlib-ng with other zlib forks.

Reply:
Actually it looks to only be exported from zlib-ng.map anyways and not zlib.map.

Reply:
`zlib-ng.map` has `zng_zcalloc` and `zlib.map` has `zcalloc`... I haven't checked when that extra `z` after `_` was dropped and why...

Reply:
Yeah but only zlib-ng.map exports `zlibng_string`. zlib.map doesn't export such a string.

Reply:
We need to include it in both, it does not actually need to be exported, it only needs to end up in the binary, and the name it has is not important at all.
Appending "Native API" or "Compat API" to that string depending on compiled version could be an interesting addition.

Reply:
Ok I have updated the strings. Let me know if you like it or not.